### PR TITLE
Added 'accessibility id' and redirecting it to 'name'

### PR DESCRIPTION
**Pull request by simongranger**

### CODE REVIEW

File: lib/commands/find.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ commands.findElOrEls = async function (strategy, selector, mult, context) {
       case "name":
         commandObject.name = `getElement${ext}ByName`;
         break;
+      case "accessibility id":
+        commandObject.name = `getElement${ext}ByName`;
```
[the remark this answers is not in the document]
Comment (simongranger) [reply lib/commands/find.js]: Right, seems like accessibility ID is redirected to `name` for iOS and for `content-desc` for Android, so we'll redirect it to `name` (what I had done in my hackday)

[the remark this answers is not in the document]
Comment (simongranger) [reply lib/commands/find.js]: Making a change in Appium Desktop would confuse iOS users....

[the remark this answers is not in the document]
Comment (TamimiGitHub) [reply lib/commands/find.js]: I agree, I was a bit reluctant as well to make the changes on the Appium Desktop.

**Comment by TamimiGitHub** [comment]
LGTM

**Pull request merged**